proxy/logging/LogFile.h

```diff
@@ -177,7 +92,12 @@ class LogFile : public LogBufferSink, public RefCountObj
   off_t
   get_size_bytes() const
   {
-    return m_file_format != LOG_FILE_PIPE ? m_bytes_written : 0;
+    if (m_file_format == LOG_FILE_PIPE)
+      return 0;
+    else if (m_log)
+      return m_log->get_size_bytes();
+    else
+      return 0;
```

Comment:
`return m_file_format == LOG_PIPE_FILE ? 0 : m_log ? m_log->get_size_bytes() : 0;`

Maybe. That's just my personal style.